sql/core/src/main/scala/org/apache/spark/sql/sources/interfaces.scala

```diff
@@ -308,7 +305,6 @@ trait InsertableRelation {
  *
  * @since 1.3.0
  */
-@Experimental
 @Unstable
```

Comment:
does it make sense to have it Unstable? It  has not changed for over 2 years...

Reply:
Agree, I wasn't sure about removing Unstable, but seems like some that clearly can go. Maybe anything not changed in a long time as you say.